external/fv3fit/fv3fit/emulation/models/microphysics.py

```diff
@@ -47,10 +49,26 @@ class MicrophysicsConfig:
     selection_map: Mapping[str, slice] = dataclasses.field(default_factory=dict)
     tendency_outputs: Mapping[str, str] = dataclasses.field(default_factory=dict)
     timestep_increment_sec: int = 900
+    enforce_positive: bool = True
 
     @classmethod
     def from_dict(cls, dict_) -> "MicrophysicsConfig":
-        return dacite.from_dict(cls, dict_, dacite.Config(strict=True))
```

Comment:
This special handling of the slice data types suggests that the user config is perhaps not lining up well with what the code implements. Can you modify the config to avoid the need for code like this?